### PR TITLE
Feature/appcli 99 get service statuses

### PR DESCRIPTION
**Pull request by ty-salter-bsl**

### CODE REVIEW

File: appcli/commands/service_cli.py [hunk 1]
```diff
@@ -239,6 +256,11 @@ def _action_orchestrator(
             pre_hook = hooks.pre_shutdown
             post_hook = hooks.post_shutdown
 
+        elif action == ServiceAction.STATUS:
+            action_run_function = self.orchestrator.status
```
Comment (benjamin-wiffen) [inline appcli/commands/service_cli.py:260]: Do we want to consider writing pre/post hooks for `STATUS` calls?

File: appcli/commands/service_cli.py [hunk 2]
```diff
@@ -43,6 +43,7 @@ class ServiceAction(enum.Enum):
 
     START = enum.auto()
     SHUTDOWN = enum.auto()
+    STATUS = enum.auto()
```
Comment (thomas-anderson-bsl) [inline appcli/commands/service_cli.py:46]: Missing from the python doc above

File: appcli/commands/service_cli.py (hunk 1 above)
Comment (thomas-anderson-bsl) [reply appcli/commands/service_cli.py:260]: Nah - it's just a snapshot look at what's running, not a lifecycle event

File: appcli/commands/service_cli.py [hunk 3]
```diff
@@ -253,14 +275,17 @@ def _action_orchestrator(
...
         logger.info(pre_run_log_message)
         result = action_run_function(ctx.obj, service_names)
 
-        logger.debug(f"Running post-{action.name} hook")
-        post_hook(ctx, result)
+        if post_hook is not None:
+            logger.debug(f"Running post-{action.name} hook")
+            post_hook(ctx, result)
 
+        click.echo(result.stdout)
```
Comment (thomas-anderson-bsl) [inline appcli/commands/service_cli.py:289]: What's the purpose of this line, and do we need it?

There may be some implications of echoing to stdout. I'm not certain, but there might be issues with custom_commands that use stdout

**Review by thomas-anderson-bsl — COMMENTED** [review]
Few minor comments

File: appcli/commands/service_cli.py (hunk 3 above)
Comment (thomas-anderson-bsl) [reply appcli/commands/service_cli.py:289]: Actually I think the issue I'm thinking of is unfounded.
In some of our projects that use appcli, the `result` stdout is captured when we do `orchestrator.exec`, but here we're calling the click command which is outside that call stack, so I'm not concerned.

**Pull request merged**